lib/internal/fs/utils.js

```diff
@@ -112,15 +112,15 @@ for (const name of Reflect.ownKeys(Dirent.prototype)) {
 }
 
 function copyObject(source) {
```

Comment:
What about Object.assign ?

Reply:
We’d probably want to benchmark that first

Reply:
It's not likely to be much slower than `for-in` to be honest. /cc @bmeurer

Reply:
Is this even a bit faster ?

Reply:
Recent versions of V8 should have fast `Object.assign()`.